bigtable/client/internal/unary_rpc_utils.h

```diff
@@ -163,11 +165,11 @@ struct UnaryRpcUtils {
       std::unique_ptr<bigtable::RPCBackoffPolicy> backoff_policy,
       bigtable::MetadataUpdatePolicy const& metadata_update_policy,
       MemberFunction function,
-      typename CheckSignature<MemberFunction>::RequestType const& request,
-      char const* error_message) {
+     typename CheckSignature<MemberFunction>::RequestType const &request,
```

Comment:
You are reverting the changes from #352, please use the defined alignment for `&` and `*`.